### DISCUSSION TITLE
Azure AD Only

### DISCUSSION THREAD

**Discussion by integyjc** [Azure/avdaccelerator#327] [Q&A]
I may be missing something here, is there a Cloud only (Azure AD Only) version of this accelerator?

**Comment by danycontre** [comment]
@integyjc missed this one, yes you have the option to set the identity service provider as Azure AD.

Parameter is avdIdentityServiceProvider

In the portal UI the option is under identity tab: 
<img width="595" alt="image" src="https://user-images.githubusercontent.com/78437433/234084254-f3548487-1692-4f4a-8cdd-5dd3d2ef285f.png">
